rp2040-hal/src/gpio/mod.rs

```diff
@@ -62,3 +62,55 @@ pub enum OutputSlewRate {
     /// Slew fast
     Fast,
 }
+
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
+/// Interrupt override state.
+pub enum InterruptOverride {
+    /// Don't invert the interrupt.
+    DontInvert = 0,
+    /// Invert the interrupt.
+    Invert = 1,
+    /// Drive interrupt low.
+    AlwaysLow = 2,
+    /// Drive interrupt high.
+    AlwaysHigh = 3,
+}
+
+#[derive(Clone, Copy, Eq, PartialEq, Debug)]
+/// Input override state.
+pub enum InputOverride {
+    /// Don't invert the peripheral input.
+    DontInvert = 0,
+    /// Invert the peripheral input.
+    Invert = 1,
+    /// Drive peripheral output low.
```

Comment:
I assume these will drive the input low, not the output?

Reply:
That's also how I understand it.